pkgs/development/libraries/onnxruntime/nvcc-gsl.patch

```diff
@@ -1,8 +1,8 @@
 diff --git a/cmake/external/onnxruntime_external_deps.cmake b/cmake/external/onnxruntime_external_deps.cmake
-index 9effd1a2db..faff5e8de7 100644
+index 775576a..ccea13c 100644
```

Comment:
i think this patch might be unnecessary, given the comment in `onnxruntime/default.nix` on the line where this patch is included